CONTRIBUTING.md

```diff
@@ -1 +1,25 @@
 # Developer Guide
+
+Thank you for taking the time out to contribute to project Theia!
+Project Theia follows the [Antrea Developer Guide]
```

Comment:
Question - why just for two cases, you define the link at the end of the doc? To be consistent, maybe just add the url inline after the text.

Reply:
Yes, I added a link at the end of the doc. There are three [Antrea Developer Guide]. I was thinking not adding long url inline for duplicated cases.
But for consistency, I agree we just add the url inline.
Fixed. Thanks!

Reply:
Oh, I did not see you have three instances! Then it might make sense to define the link separately. Probably we should just check the Antrea CI link in ci/README

Reply:
Would let you decide whether to revert back to the original way. Sorry for the confusion.

Reply:
I think we can use current settings. Maybe we should use links at the end of doc when there are multiple cases that have duplicates?

Reply:
You do have three references to "Antrea Developer Guide" in this doc, right.

Reply:
Right. Do you think it's better to revert it?

Reply:
Yeah, I prefer your previous version.

Reply:
Updated. Thanks!